content/200-orm/400-tools/06-prisma-studio.mdx

```diff
@@ -23,7 +23,13 @@ Run `npx prisma studio` in your terminal.
 
 ## Models (tables or collections)
 
-When you first open Prisma Studio, you see a list of all models defined in your Prisma schema file.
+When you first open Prisma Studio, you will see a data table layout with a sidebar showing a list of all models defined in your Prisma schema file.
+
+<img
```

Comment:
If possible I’d really prefer us to move away from this and instead use the markdown image syntax. I think we had this because of the need to specify the specific width, but if we don’t have to we shouldn’t.